### ISSUE TITLE
[0.12-rc2] Outdated tutorial on two-column document styling

### ISSUE THREAD

**Issue by jarnowic** [typst/typst#5215]
### Description

Consider the following input, taken almost verbatim from the Advanced Styling Tutorial section:

```
#set page(
  paper: "us-letter",
  header: align(right)[
    A fluid dynamic model for
    glacier flow
  ],
  numbering: "1",
)
#set par(justify: true)
#set text(
  font: "Palatino Linotype",
  size: 11pt,
)

#align(center, text(17pt)[
  *A fluid dynamic model
  for glacier flow*
])

#grid(
  columns: (1fr, 1fr),
  align(center)[
    Therese Tungsten \
    Artos Institute \
    #link("mailto:tung@artos.edu")
  ],
  align(center)[
    Dr. John Doe \
    Artos Institute \
    #link("mailto:doe@artos.edu")
  ]
)

#align(center)[
  #set par(justify: false)
  *Abstract* \
  #lorem(80)
]

#show: rest => columns(2, rest)

= Introduction
#lorem(60),#footnote[#lorem(32)] #lorem(20).#footnote[#lorem(40)]


= Related Work

#lorem(60).#footnote[#lorem(20)]

```

With typst <0.12, the footnotes are typeset in different columns: 

![screenshot 2024-10-15_19-07-05](https://github.com/user-attachments/assets/3dce4d71-6c27-4597-aca9-1a94712d63aa)

while with the release candidates we see:

![screenshot 2024-10-15_19-08-03](https://github.com/user-attachments/assets/72a2337b-eec6-4282-98a8-821bb46a0e49)

If I try to fix the output adding `columns:2` to `#set page()`, while commenting out `#show: rest => columns(2, rest)
`, I get the following output:

![screenshot 2024-10-15_19-11-15](https://github.com/user-attachments/assets/32434329-d7d1-445f-9fbe-b3306b6f3048)

So I fear the 0.12 series has introduced a bug in the layout algorithm of footnotes.

### Operating system

Windows

### Typst version

- [X] I am using the latest version of Typst

**Comment by PgBiel** [comment]
`#set page(columns: 2)` is indeed now the recommended way to typeset your document with two columns, using a multi-column float at the top for the title and abstract, as such:

```typ
#set page(
  paper: "us-letter",
  header: align(right)[
    A fluid dynamic model for
    glacier flow
  ],
  numbering: "1",
  columns: 2,
)
#set par(justify: true)
#set text(
  font: "Tex Gyre Pagella",
  size: 11pt,
)

#place(top, float: true, scope: "parent")[
  #align(center, text(17pt)[
    *A fluid dynamic model
    for glacier flow*
  ])
  
  #grid(
    columns: (1fr, 1fr),
    align(center)[
      Therese Tungsten \
      Artos Institute \
      #link("mailto:tung@artos.edu")
    ],
    align(center)[
      Dr. John Doe \
      Artos Institute \
      #link("mailto:doe@artos.edu")
    ]
  )
  
  #align(center)[
    #set par(justify: false)
    *Abstract* \
    #lorem(80)
  ]
]

= Introduction
#lorem(60),#footnote[#lorem(32)] #lorem(20).#footnote[#lorem(40)]


= Related Work

#lorem(60).#footnote[#lorem(20)]
```

<details><summary>Output</summary>
<p>

![output: one footnote section per column](https://github.com/user-attachments/assets/1d1ee00c-43ef-40f6-8e7a-9b7f28f4e012)

</p>
</details> 

You're right that the tutorial needs to be updated. cc @laurmaedje

**Comment by jarnowic** [comment]
it seems that the `scope: ...` parameter of `place` is not yet documented either: it yields an error in typst 0.11 .

**Comment by laurmaedje** [comment]
It is new in 0.12 and documented on staging.typst.app/docs (since 0.12 is not out yet).

**Comment by PgBiel** [comment]
Fixed by #5228
